package.json

```diff
@@ -159,6 +160,7 @@
     "jest-watch-typeahead": "^2.2.2",
     "mini-css-extract-plugin": "^2.7.6",
     "msw": "^1.3.1",
+    "prettier": "3.0.3",
```

Comment:
We probably want this one to be `^3.0.3` (running `yarn` again after the changes may be required to update the lock file).